Трохимец Сергий/First HomeWork/Task1.rb

```diff
@@ -0,0 +1,13 @@
+# Task 1
+array = Array.new(3){ Array.new(3){rand(-10..10)}}.flatten # рандомний двовимірний масив (3 на 3)
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#spaces-operators
It's better to add spaces so every part of your code is stand out visually

```suggestion
array = Array.new(3){ Array.new(3){ rand(-10..10) } }.flatten # рандомний двовимірний масив (3 на 3)
```